### PR TITLE
CLOUDP-272605 Remove IDPs on cleanup script

### PR DESCRIPTION
**Pull request by fmenezes**
## Proposed changes
Remove IDPs on cleanup script

_Jira ticket:_ CLOUDP-272605

Closes #[issue number]

## Checklist

- [ ] I have signed the [MongoDB CLA](https://www.mongodb.com/legal/contributor-agreement)
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have added any necessary documentation in document requirements section listed in [CONTRIBUTING.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/CONTRIBUTING.md) (if appropriate)
- [ ] I have addressed the @mongodb/docs-cloud-team comments (if appropriate)
- [ ] I have updated [test/README.md](https://github.com/mongodb/mongodb-atlas-cli/blob/master/test/README.md) (if an e2e test has been added)
- [ ] I have run `make fmt` and formatted my code

### CODE REVIEW

**Review by jeroenvervaeke — APPROVED** [review]

**Review by blva — APPROVED** [review]

File: test/e2e/atlas/helper_test.go [hunk 1]
```diff
@@ -1265,3 +1265,52 @@ func installExamplePlugin(t *testing.T, cliPath string, version string) {
...
+	resp, err := e2e.RunAndGetStdOut(cmd)
+	require.NoError(t, err, string(resp))
+	var idps *atlasv2.PaginatedFederationIdentityProvider
+	require.NoError(t, json.Unmarshal(resp, &idps))
+	return idps
+}
+
+func deleteIDP(t *testing.T, cliPath string, id string, fedSettingsID string) {
+	t.Helper()
+	cmd := exec.Command(cliPath, federatedAuthenticationEntity, federationSettingsEntity, "identityProvider", "delete", id, "--federationSettingsId", fedSettingsID, "--force")
```
Comment (blva) [inline test/e2e/atlas/helper_test.go:1303]: did we test this works? I think sometimes you need to check if they are linked to orgs and need to unlink first?

Comment (fmenezes) [reply test/e2e/atlas/helper_test.go:1303]: It worked on my test yes, not too sure about the edge case you've mentioned, my advice is let's see if we catch the bug if so we open a new jira

Comment (blva) [reply test/e2e/atlas/helper_test.go:1303]: makes sense!
